MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/Tomography/TomographyIfaceModel.h

```diff
@@ -83,9 +84,56 @@ class MANTIDQT_CUSTOMINTERFACES_DLL TomographyIfaceModel {
 
   /// Username last logged in, if any.
   std::string loggedIn() const { return m_loggedInUser; }
+
   // TODO: add companion currentComputeResource where LoggedIn() is in
-  void usingTool(const std::string &tool) { m_currentTool = tool; }
-  std::string usingTool() const { return m_currentTool; }
+  //--------------------------------------------
+  // Current tool Settings
+  //--------------------------------------------
+  void usingTool(const std::string &tool) { m_currentToolName = tool; }
+  std::string usingTool() const { return m_currentToolName; }
+
+  void setCurrentToolMethod(std::string toolMethod) {
+    m_currentToolMethod = toolMethod;
+  }
+  std::string getCurrentToolMethod() const { return m_currentToolMethod; }
+
+  void setCurrentToolSettings(std::shared_ptr<TomoRecToolConfig> settings) {
+    m_currentToolSettings = settings;
+  }
+
+  TomoRecToolConfig *getCurrentToolSettings() const {
+    return m_currentToolSettings.get();
+  }
+
+  //--------------------------------------------
+  // Access to the system settings information
+  //--------------------------------------------
+  // get the remote scripts base dir from the system settings
+  std::string getCurrentRemoteScriptsBasePath() const {
+    return m_systemSettings.m_remote.m_basePathReconScripts;
+  }
+
+  /// get the local paths from the system settings
+  std::string getCurrentLocalScriptsBasePath() const {
+    return m_systemSettings.m_local.m_reconScriptsPath;
+  }
+
+  std::string getExeternalInterpreterPath() const {
+    return m_systemSettings.m_local.m_externalInterpreterPath;
+  }
+
+  /// get the experiment reference currently selected
+  std::string getCurrentExperimentReference() const {
+    return m_systemSettings.m_experimentReference;
+  }
+
+  /// returns the scripts folder, used for remote path
+  std::string getTomoScriptFolderPath() const { return g_tomoScriptFolderPath; }
+
+  /// returns the tomo script location path inside the scripts folder
+  std::string getTomoScriptLocationPath() const {
+    return g_mainReconstructionScript;
+  }
 
   /// ping the (remote) compute resource
   bool doPing(const std::string &compRes);
```

Comment:
Having the `do` fits with how the rest of the methods are named in the gui classes, having the `can` would be a one off deviation and I don't think it's worth it, if anything I think just renaming it to `ping` would be nice, but I'd like to leave it as it is